kge/config.py

```diff
@@ -238,22 +238,27 @@ def set(
     def _import(self, module_name: str):
         """Imports the specified module configuration.
 
-        Adds the configuration options from kge/model/<module_name>.yaml to
+        Adds the configuration options from <module_name>.yaml to
         the configuration. Retains existing module configurations, but verifies
         that fields and their types are correct.
 
         """
-        import kge.model, kge.model.embedder
         from kge.misc import filename_in_module
+        import importlib
+        module_names = self.get_default("modules")
+        modules = [importlib.import_module(m) for m in module_names]
```

Comment:
This is a quite frequent piece of code. Perhaps have a `get_modules()` method in `Config`?

Reply:
I did make this a method, although I only ended up calling it once because of the way dealt with the other duplicate code you pointed out.